src/OpenSEE/Scripts/TSX/defaults.ts

```diff
@@ -245,10 +245,9 @@ export const defaultSettings = {
     TimeUnit: {
         current: 5, options: [
```

Comment:
I believe this will cause issues if `current` is `5` but `options[5]` does not exist.

Reply:
This is also confusing because the PR was branched from an earlier commit. It's why there are conflicts in `defaults.ts`. Try pulling the latest code and then applying your changes to that.

Reply:
Ok, Going to go ahead and delete this branch/pull request so I can redo the changes on the latest version of the master branch.